classic_ml/README.md

```diff
@@ -0,0 +1,80 @@
+# Classical ML Models
+
+This folder contains the script to run Traditional ML models from scikit-learn using onnx framework.
+
+Each model has been trained on specific dataset and it should be tested using only that specific dataset. For ex Regression models are using diabetes dataset and classification models are using iris dataset.
+
+Dataset can be downloaded from here:
+```
+https://ampereaimodelzoo.s3.eu-central-1.amazonaws.com/sklearn2onnx/*.csv
+
+```
+
+### Metrics
+
+Linear Regression
+
+ ```                             mean  /      median  / 90th-percentile
+ Latency           [ms]:       0.01  /        0.01  /        0.01
+ Throughput [samples/s]:   80442.76  /    91180.52  /    82241.25
+ ```
+
+Logistic Regression
+
+ ```                              mean  /      median  / 90th-percentile
+ Latency           [ms]:       0.02  /        0.02  /        0.02
+ Throughput [samples/s]:   56281.19  /    61680.94  /    55924.05
+ ```
+
+DecisionTreeRegressor
+
+ ```                            mean  /      median  / 90th-percentile
+ Latency           [ms]:       0.01  /        0.01  /        0.01
+ Throughput [samples/s]:   82953.00  /    91180.52  /    82241.25
+ ```
+DecisionTreeClassifier
+
+ ```                             mean  /      median  / 90th-percentile
+ Latency           [ms]:       0.02  /        0.01  /        0.02
+ Throughput [samples/s]:   58106.38  /    67650.06  /    61680.94
+ ```
+
+### model
+
+Model description can be found here : https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html for all the models.
+
+Models can be downloaded from here:
+```
+ https://ampereaimodelzoo.s3.eu-central-1.amazonaws.com/sklearn2onnx/
+
+```
+Link for particular model is for ex: 
+```
+https://ampereaimodelzoo.s3.eu-central-1.amazonaws.com/sklearn2onnx/DecisionTreeRegressor_model.onnx
```

Comment:
Could we list all models' urls + corresponding datasets?

Reply:
sure.